book/src/lib/lib.rs

```diff
@@ -1,5 +1,7 @@
 pub mod db;
 mod rpc;
+#[cfg(feature = "test-data")]
```

Comment:
Feature name?

Reply:
the "feature" is fixed syntax. I can't rename that.

Reply:
I know that, this was related to the `test-data`.